portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,47 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
```

Comment:
Sorry I may have missed... Is it called somewhere?

Reply:
Nope, sorry I forgot to delete it! It came with the tutorial and I ended up not using it.